.github/workflows/release-series-images.yml

```diff
@@ -95,33 +80,31 @@ jobs:
         # which is required for build-push-action
         IFS=,
         echo "::set-output name=taglist::${tag_list[*]}"
-
     - name: Set up Docker Buildx
       uses: docker/setup-buildx-action@v1
-
-    - name: Log in to Docker Hub
-      uses: docker/login-action@v1
-      if: github.event_name != 'pull_request' && startsWith(github.repository, 'opensciencegrid/')
       with:
-        username: ${{ secrets.DOCKER_USERNAME }}
-        password: ${{ secrets.DOCKER_PASSWORD }}
-
-    - name: Log in to OSG Harbor
-      uses: docker/login-action@v1
-      if: github.event_name != 'pull_request' && startsWith(github.repository, 'opensciencegrid/')
+        driver: docker
+    
+    - name: Build and push Hosted-CE Docker image
+      uses: docker/build-push-action@v2.2.2
       with:
-        registry: hub.opensciencegrid.org
-        username: ${{ secrets.OSG_HARBOR_ROBOT_USER }}
-        password: ${{ secrets.OSG_HARBOR_ROBOT_PASSWORD }}
-
-    - name: Build and push Docker image
+        load: true
+        push: ${{ github.event_name != 'pull_request' && startsWith(github.repository, 'opensciencegrid/') }}
+        context: ./hosted-ce
+        file: ./hosted-ce/Dockerfile
+        build-args: |
+          BASE_YUM_REPO=${{ matrix.repo }}
+          BASE_OSG_SERIES=${{ matrix.osg_series }}
+        tags: "${{ steps.generate-tag-list.outputs.taglist }}"
+    
+    - name: Build and push osg-ce-condor Docker image
       uses: docker/build-push-action@v2.2.2
       with:
+        load: true
         push: ${{ github.event_name != 'pull_request' && startsWith(github.repository, 'opensciencegrid/') }}
-        context: .
+        context: ./osg-ce-condor
+        file: ./osg-ce-condor/Dockerfile
         build-args: |
           BASE_YUM_REPO=${{ matrix.repo }}
           BASE_OSG_SERIES=${{ matrix.osg_series }}
         tags: "${{ steps.generate-tag-list.outputs.taglist }}"
-        target: ${{ matrix.image }}
-        cache-from: type=local,src=/tmp/.buildx-cache
```

Comment:
We shouldn't have /both/ these steps because it means that we'll build both images for every single job, i.e. the hosted-ce build jobs will build and push hosted-ce AND osg-ce-condor images